app/helpers/course/discussion/topics_helper.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+module Course::Discussion::TopicsHelper
+  # Display code lines in file.
+  #
+  # @param [Course::Assessment::Answer::ProgrammingFile] file The code file.
+  # @param [Integer] line_start The one based start line number.
```

Comment:
Currently in our repo we have both `Fixnum` and `Integer` in the documentation, maybe we should standardise that in a housekeeping PR. 

I personally prefer `Integer` because it's the parent of `Bignum` and `Fixnum`, which is more general.